packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -76,7 +76,13 @@ export const messages = {
   force_apex_test_run_all_tests_description_text:
     'Runs all tests in the current project',
 
+  force_apex_test_run_class_text: 'SFDX: Invoke All Apex Tests in Class',
+  force_apex_test_run_method_text: 'SFDX: Invoke Apex Test Method',
   force_apex_test_run_codeAction_description_text: 'Run Apex test(s)',
+  force_apex_test_run_codeAction_no_class_test_param_text:
+    'Test class not provided.  Please run Apex test code action on a test class.',
+  force_apex_test_run_codeAction_no_method_test_param_text:
+    'Test method not provided.  Please run Apex test code action on a test method.',
```

Comment:
@ruthemmanuelle, see changes above.  Thanks.

Reply:
1. We use one space (not two) between sentences.
2. You don't need to say "please." Our style guidelines specify that we should save "please" for when we're asking the user to do something particularly inconvenient.
3. Just in case, let's say why the class and method were unacceptable.

'Test class not provided. Run the code action on a class annotated with @isTest.'
'Test method not provided. Run the code action on a method annotated with @isTest or testMethod.'